cli/fmt.rs

```diff
@@ -21,7 +21,7 @@ use std::io::Write;
 use std::path::Path;
 use std::path::PathBuf;
 use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
-use std::sync::{Arc, Mutex};
```

Comment:
@bartlomieju you may want to update the `deno_lint` example to remove the `Arc`s. I just remembered this isn't necessary with Rayon.